pkgs/kde/misc/klevernotes/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  lib,
+  mkKdeDerivation,
+  fetchurl,
+  kio,
+  qtsvg,
+  qtwebengine,
+}:
+mkKdeDerivation rec {
+  pname = "klevernotes";
+  version = "1.0.0";
+
+  src = fetchurl {
+    url = "mirror://kde/stable/klevernotes/${version}/klevernotes-${version}.tar.xz";
+    hash = "sha256-9nXOaa2/DyJO3nXGbnvPaYjXFrVoAgx52th67l5geOU=";
+  };
+
+  extraBuildInputs = [
+    kio
+    qtsvg
+    qtwebengine
+  ];
+
+  meta.license = [ lib.licenses.gpl3Plus ];
```

Comment:
License is definitely not just that: https://invent.kde.org/office/klevernotes/-/tree/master/LICENSES?ref_type=heads

Reply:
My bad, I didn’t see this part of the repo and took the license from the product homepage only. I added most of them, except the FSFAP which it seems isn’t in `lib.licenses`. Should I create another MR which adds it there and then here, or simply define it locally?